qa/tasks/cephfs/test_volumes.py

```diff
@@ -2530,3 +2530,212 @@ def test_subvolume_snapshot_clone_cancel_pending(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+    def test_subvolume_ops_on_nonexistent_vol(self):
```

Comment:
nit: have a single test case that performs operations on non-existent volume

Reply:
Done